.gitignore

```diff
@@ -1,151 +1,24 @@
-# Created by http://www.gitignore.io
```

Comment:
Good suggestions! The new .gitignore file is a simplified version of the "official" file from github/gitignore. - The main issue with some predefined gitignore files is that they declare TONS of unnecessary and redundant stuff. Like 90% of the lines were never used. You're right that some of the additions are in fact unnecessary, for example the google-services.json file (your project does not use Firebase). I have removed it and amended the commit.